modules/troubleshooting-common-problems-with-distributed-workloads-for-administrators.adoc

```diff
@@ -0,0 +1,286 @@
+
+. Define a default local queue.
++
+ifndef::upstream[]
+For information about creating a local queue and defining a default local queue, see link:{rhoaidocshome}{default-format-url}/working_with_distributed_workloads/configuring-distributed-workloads_distributed-workloads#configuring-quota-management-for-distributed-workloads_distributed-workloads[Configuring quota management for distributed workloads].
+endif::[]
+ifdef::upstream[]
+For information about configuring resource flavors and quotas, see link:{odhdocshome}/working-with-distributed-workloads/#configuring-quota-management-for-distributed-workloads_distributed-workloads[Configuring quota management for distributed workloads].
+endif::[]
+
+ 
+== A user receives a *local_queue provided does not exist* error message
+
+.Problem
+After the user runs the `cluster.up()` command, the following error is shown:
+
+[source,bash]
+----
+local_queue provided does not exist or is not in this namespace. Please provide the correct local_queue name in Cluster Configuration.
+----
+
+.Diagnosis
+An incorrect value is specified for the local queue in the cluster configuration, or an incorrect default local queue is defined.
+The specified local queue either does not exist, or exists in a different namespace.
+
+.Resolution
+ifdef::upstream,self-managed[]
+.. In the {openshift-platform} console, select the user's project from the *Project* list. 
+endif::[]
+ifdef::cloud-service[]
+..  In the OpenShift console, select the user's project from the *Project* list.
+endif::[]
+. Click *Search*, and from the *Resources* list, select *LocalQueue*.
+. Resolve the problem in one of the following ways:
+. If no local queues are found, create a local queue.
+. Provide the user with the details of the local queues in their project.
+Advise the user to ensure that they spelled the local queue name correctly in their cluster configuration, and that the `namespace` value in the cluster configuration matches their project name.
+If the user does not specify a `namespace` value in the cluster configuration, the Ray cluster is created in the current project.
+
+. Define a default local queue.
++
+ifndef::upstream[]
+For information about creating a local queue and defining a default local queue, see link:{rhoaidocshome}{default-format-url}/working_with_distributed_workloads/configuring-distributed-workloads_distributed-workloads#configuring-quota-management-for-distributed-workloads_distributed-workloads[Configuring quota management for distributed workloads].
+endif::[]
+ifdef::upstream[]
+For information about configuring resource flavors and quotas, see link:{odhdocshome}/working-with-distributed-workloads/#configuring-quota-management-for-distributed-workloads_distributed-workloads[Configuring quota management for distributed workloads].
+endif::[]
+
+
+== A user cannot create a Ray cluster or submit jobs
+
+.Problem
+After the user runs the `cluster.up()` command, an error similar to the following error is shown:
```

Comment:
```suggestion
After the user runs the `cluster.up()` command, an error similar to the following is shown:
```
Nitpicky - just makes the sentence a little less repetitive.

Reply:
IBM Style says "always use _following_ as an adjective before a noun" so I changed the second occurrence to "text" to avoid the repetition :)